app/api/enhance/route.ts

```diff
@@ -0,0 +1,44 @@
+import {NextRequest, NextResponse} from 'next/server'
+import {OpenAI} from 'openai'
+
+const openai = new OpenAI({
```

Comment:
This line should be moved to the top of the route handler function. That should resolve the build issue. 

Also, we don't need to pass in the API key explicitly as long as you have the variable in your `.env.local`.

Reply:
On it!

Reply:
Done!